src/node_api.cc

```diff
@@ -3401,15 +3401,22 @@ napi_status napi_delete_async_work(napi_env env, napi_async_work work) {
   return napi_clear_last_error(env);
 }
 
+napi_status napi_get_uv_event_loop(napi_env env, uv_loop_t** loop) {
+  CHECK_ENV(env);
+  CHECK_ARG(env, loop);
+  *loop = node::GetCurrentEventLoop(env->isolate);
```

Comment:
If you make that change, consider adding a sanity check here that `loop != nullptr`.

Reply:
I’ve moved the `GetCurrentEventLoop` call to the `napi_env` initialization code, I think that should be fine.